Makefile

```diff
@@ -0,0 +1,40 @@
+# This Makefile is based on the Makefile defined in the Python Best Practices repository:
+# https://git.datapunt.amsterdam.nl/Datapunt/python-best-practices/blob/master/dependency_management/
+#
+# VERSION = 2020.01.29
+.PHONY: app manifests
+
+NAME = $*
+
+export API_PORT=31410
+export AUTH_PORT=31430
+export IMAGE_PORT_API=31450
+export ADMIN_PORT=31470
+export CMS_PORT=31490
```

Comment:
You want to define the ports here instead of in the docker compose?

Reply:
Yes - this docker compose comes from upstream, and actually I don't want to change it. Still had to do that for the `image:` lines, but this is something I actually still want to change later, in something that can be pushes upstream.